### DISCUSSION TITLE
Is it possible to line-break the labels of Xaxis in the chart

### DISCUSSION THREAD

**Discussion by Dongseung** [nhn/tui.chart#748] [Q&A]
I want to set the labels of xAxis as 2 lines.
I have used formatter but it does not work at all or show me as 'undefined'

Can I set the labels as 2 lines  such as :
ex) : " day
          date"
![image](https://user-images.githubusercontent.com/75294699/174247807-bca0737c-2ed5-4ae0-a74a-87353597dbb1.png)

**Comment by jwlee1108** [comment]
@Dongseung Sorry for the late response.
I think you can use [`formatter`](https://github.com/nhn/tui.chart/blob/main/docs/en/common-axes.md#formatter) feature.

**Reply by Dongseung** [reply]
@jwlee1108 
Thank you for the answers.

However, I have used formatter feature such as

```
categories: [`요일/월요일`],

xAxis: {
        align: 'center',
        title: '',
        label: {
           formatter: (value) => {
             const newDate = value.split('/')
             return newDate[0] + '<br/>'  + newDate[1];
           },
        },
      },
  
```
  but it does not work...
